link/kprobe_multi_test.go

```diff
@@ -106,9 +106,9 @@ func TestKprobeMultiProgramCall(t *testing.T) {
 
 	// Trigger ebpf program call.
 	unix.Getpid()
+	unix.Gettid()
 
-	// Assert that the value at index 0 has been updated to 1.
-	assertMapValue(t, m, 0, 1)
+	assertMapValueGE(t, m, 0, 2)
```

Comment:
right, that would indicate we got hit from both probes.. but we'd need to call the helper to get the cookie value, which might not be supported on older kernels, but yes it's doable.. any chance this could be a follow up? ;-)

Reply:
> we'd need to call the helper to get the cookie value, which might not be supported on older kernels

Since we're testing kprobe-multi anyways, don't those kernels have cookie support already?